lib/domgen/jpa/model.rb

```diff
@@ -702,7 +710,8 @@ def pre_verify
             end
           end
           if jpql
-            jpql = "(#{jpql}) AND (#{self.default_jpql_criterion})" if self.default_jpql_criterion
+            jpql = "(#{jpql})"
```

Comment:
Unfortunately putting brackets around jpql will not always work in all circumstances. If `self.default_jpql_criterion` is nil you should not modify the jpql variable.